src/ansys/additive/core/microstructure.py

```diff
@@ -558,57 +540,69 @@ def input(self):
 
     @property
     def xy_vtk(self) -> str:
-        """Path to the VTK file containing the 2-D grain structure data in the XY plane."""
-        return self._xy_vtk
+        """Path to the VTK file containing the 2D grain structure data in the XY plane.
+
+        The VTK file contains scalar data sets `GrainBoundaries`, `Orientation_(deg)` and
+        `GrainNumber`.
```

Comment:
```suggestion
        The VTK file contains these scalar data sets: ``GrainBoundaries``, ``Orientation_(deg)``, and
        ``GrainNumber``.
```
Single backticks result in italic formatting, while double backticks result in code entity formatting. We always use an Oxford comma, which is simply a comma before an "and" or an "or" in a list of more than two items.